Service/Request/Visitor/EndpointConfiguration.php

```diff
@@ -0,0 +1,58 @@
+<?php
+declare(strict_types=1);
```

Comment:
I think, here CS warning — here should be one empty line

Reply:
New line added